PIPELINE: rescaling for rest of the pipelines

UdomkarnBoonyaprasert opened the pull request:
- add resize function call from compNumpyTransforamtion in the rest of the pipelines (abdomen already has this)
- all pipelines except for abdomen pipeline now has resolution limit of 256
- add logging info for marching cube (time taken and face count)
- minor fix with dicom rotation

This is to address issue from #47.

UdomkarnBoonyaprasert committed:
PIPELINE: add rescaling for rest of the pipelines
- add resize function call from compNumpyTransforamtion in the rest of the pipelines (abdomen already has this)
- add logging info for marching cube (time taken and face count)
- minor fix with dicom rotation

nielsboecker commented on HoloPipelines/pipelines/components/compDicom2numpy.py:
```diff
@@ -88,7 +88,7 @@ def reorientateNumpyList(numpyList):
     numpyList = numpyList.transpose(2, 1, 0)
     numpyList = np.flip(numpyList, 0)
-    numpyList = np.flip(numpyList, 1)
+    # numpyList = np.flip(numpyList, 1) # need to confirm this
```
Can you confirm this today as you will be unavailable for the next 6 days? Otherwise it's hard to accept this PR

UdomkarnBoonyaprasert replied:
Confirmed, NumPy array is now in correct orientation. Commented code removed

nielsboecker replied:
thanks

nielsboecker commented on HoloPipelines/pipelines/pipelineDicom2glb.py:
```diff
@@ -2,17 +2,21 @@
 
 def main(dicomFolderPath, outputGlbPath, threshold):
     generatedNumpyList = compDicom2numpy.main(str(pathlib.Path(dicomFolderPath)))
+    resizedNumpyList = compNumpyTransformation.sizeLimit(generatedNumpyList, limit=256)
```
if 256 is limit for all pipelines, or at least the default, maybe we can bring it to the shared configuration file?

nielsboecker replied:
How did you decide for 256 for all others, and why does the abdomen pipeline have a different value?

UdomkarnBoonyaprasert replied:
I added a `config.ini` in the base level of HoloPipeline. At the moment it only holds default value for resolution limit. This value is called as default value in `compNumpyTransformation`

Resolution limit of abdomen pipeline is now set to default (256). I chose 256 for default as the final outputs from experiment did not show significant downgrade in quality, also that this value is approximately the median to the values I usually get from most Dicom after resampling (220 to 300). We also plan to implement fast quadratic simplification (wrapper component still in my local branch) we can expect the file to be smaller.

UdomkarnBoonyaprasert committed:
PIPELINE: add config file
- add shared config file (contain only default value for resolution limit at the moment. Can use to store default value for simplification later)
- change default resolution of abdomen pipeline to 256
- remove unused comment (compDicom2numpy)

UdomkarnBoonyaprasert committed:
PIPELINE: minor bug fix

UdomkarnBoonyaprasert committed:
PIPELINE: remove dafult resolution from lung pipeline

nbckr commented:
**Update**: Thanks @UdomkarnBoonyaprasert. As discussed, I read your changes and copied them to my branch in  80069112e7b59319214b9db305074976a3601e41 for #45. I will therefore reject this PR.
